java/ql/src/experimental/semmle/code/java/Logging.qll

```diff
@@ -0,0 +1,26 @@
+/**
+ * Provides classes and predicates for working with logger's and their components.
```

Comment:
```suggestion
 * Provides classes and predicates for working with loggers and their components.
```

"components" might be a little bit unspecific. Maybe it would be worth removing that part:
```suggestion
 * Provides classes and predicates for working with loggers.
```